src/tslint-cli.ts

```diff
@@ -207,6 +207,21 @@ const processFile = (file: string) => {
         process.exit(1);
     }
 
+    const buffer = new Buffer(256);
```

Comment:
the `Buffer` constructor doesn't initialize its content. As the file we're reading could be less than 256 bytes, we could end up reading junk data in the conditional. (Although I'm not entirely sure of the behavior of `readSync`.) I think we should zero the memory with `Buffer.fill` here